opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp

```diff
@@ -598,7 +603,7 @@ namespace Opm {
 
         const ADB tr_mult = transMult(state.pressure);
         const ADB mu    = fluidViscosity(canonicalPhaseIdx, phasePressure[canonicalPhaseIdx], state.temperature, state.rs, state.rv, cond);
-        sd_.rq[phase].mob = tr_mult * kr[canonicalPhaseIdx] / mu;
+        sd_.rq[phase].mob = tr_mult * rq[phase].kr / mu;
```

Comment:
Would it not be simpler to use `sd_.rq` here and not pass `rq` as a function argument?